recipes/bucketcache/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "BucketCache" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 28ba696fbb537c5a3ce5d7c9bb8eb5cd9cc3ace493d7d9afb299e8c5be4f9fa8 
+
+build:
+ number: 0
+ script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - decorator >=4.0.2
+    - six >=1.9.0
+    - logbook >=0.12.5
+    - represent >=1.5.1
+    - boltons
+    - python-dateutil
+    - pathlib
+
+  run:
+    - python
+    - decorator >=4.0.2
+    - six >=1.9.0
+    - logbook >=0.12.5
+    - represent >=1.5.1
```

Comment:
There are no ``logbook`` or ``represent`` package in ``conda-forge``

Reply:
Yeah, indeed, I deduced the same from the log output. Thanks! I have added the recipes to the PR.